src/test/regress/expected/anonymous_columns.out

```diff
@@ -121,7 +121,7 @@ SELECT a + 1, a - 1 FROM t0 ORDER BY 1;
         3 |        1
 (2 rows)
 
-WITH cte1 AS (SELECT row_to_json(row(a))->'f1' FROM t0) SELECT * FROM cte1 ORDER BY 1::text;
+WITH cte1 AS (SELECT row_to_json(row(a))->'f1' FROM t0) SELECT * FROM cte1 ORDER BY "?column?"::text;
```

Comment:
Tested with a column that doesn't exist in this format and got the following error as expected:

```diff
+WITH cte1 AS (SELECT row_to_json(row(a))->'f1' FROM t0) SELECT * FROM cte1 ORDER BY "?column11?"::text;
+ERROR:  column "?column11?" does not exist
+HINT:  Perhaps you meant to reference the column "cte1.?column?".
```